### ISSUE TITLE
Same log level in different channels

### ISSUE THREAD

**Issue by Smolevich** [laravel/framework#27669]
- Laravel Version: 5.7.27
- PHP Version: 7.2.14
- Database Driver & Version:

### Description:
There are two channels with same `driver=monolog` and handler (for example `StreamHandler`), but different levels (`debug`, `error`). After configuration handlers for this channels will be have same log levl debug

### Steps To Reproduce:
`config/logging.php`
```
'channels' => [
        'stack' => [
            'driver' => 'stack',
            'channels' => ['stderr', 'stdout'],
        ],
        'stderr' => [
            'driver' => 'monolog',
            'level' => 'error',
            'handler' => StreamHandler::class,
            'formatter' => env('LOG_STDERR_FORMATTER'),
            'with' => [
                'stream' => 'php://stderr',
            ],
        ],

        'stdout' => [
            'driver' => 'monolog',
            'level' => 'debug',
            'handler' => StreamHandler::class,
            'formatter' => env('LOG_STDERR_FORMATTER'),
            'with' => [
                'stream' => 'php://stdout',
            ],
        ],
    ],
```
Create console command with content in function handler `Log::debug('test', ['context' => 'test_content']);` There are two messages with same log level in output. [Link on code in framework](https://github.com/laravel/framework/blob/5.7/src/Illuminate/Log/LogManager.php#L337-L339)

**Comment by mfn** [comment]
Issue can be closed, bug is fixed in 5.8 at least.

**Comment by Smolevich** [comment]
Thanks
